site/src/examples/drawer/RightDrawer.tsx

```diff
@@ -0,0 +1,65 @@
+import { ReactElement, useState } from "react";
+
+import { Drawer, DrawerCloseButton } from "@salt-ds/lab";
+import {
+  Button,
+  FormField,
+  FormFieldHelperText,
+  FormFieldLabel,
+  Input,
+  StackLayout,
+  H2,
+  Text,
+} from "@salt-ds/core";
+
+const FormFieldExample = () => (
+  <FormField>
+    <FormFieldLabel>Label</FormFieldLabel>
+    <Input />
+    <FormFieldHelperText>Help text appears here</FormFieldHelperText>
+  </FormField>
+);
+
+export const RightDrawer = (): ReactElement => {
+  const [open, setOpen] = useState(false);
+  const id = "right-drawer";
+
+  const handleRequestOpen = () => {
+    setOpen(true);
+  };
+
+  const onOpenChange = (newOpen: boolean) => {
+    setOpen(newOpen);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  return (
+    <>
+      <Button onClick={handleRequestOpen}>Open Right Drawer</Button>
+      <Drawer
+        open={open}
+        onOpenChange={onOpenChange}
+        position="right"
+        id={id}
+        style={{ width: 500 }}
+      >
+        <StackLayout>
+          <DrawerCloseButton onClick={handleClose} />
+          <H2 id={`${id}-header`}>Section Title</H2>
```

Comment:
Why is id concat needed?

Reply:
wdym - attaching this to the title here so wanted to differentiate this the header ?